be/src/storage/tablet_schema.cpp

```diff
@@ -546,6 +662,48 @@ void TabletSchema::to_schema_pb(TabletSchemaPB* tablet_schema_pb) const {
     tablet_schema_pb->mutable_sort_key_idxes()->Add(_sort_key_idxes.begin(), _sort_key_idxes.end());
     tablet_schema_pb->mutable_sort_key_unique_ids()->Add(_sort_key_uids.begin(), _sort_key_uids.end());
     tablet_schema_pb->set_schema_version(_schema_version);
+    for (auto& index : _indexes) {
+        auto* tablet_index_pb = tablet_schema_pb->add_table_indices();
+        index.to_schema_pb(tablet_index_pb);
+    }
+}
+
+Status TabletSchema::get_indexes_for_column(int32_t col_unique_id,
+                                            std::unordered_map<IndexType, TabletIndex>* res) const {
```

Comment:
It is a little waste to have TabletIndex - the whole struct as the value type of map.

Reply:
It's more convenient for following PR to hold this mapping during building index, early refactoring will lose code coherence, it's better to refactor in next PR.